pkg/database/realm.go

```diff
@@ -207,6 +207,19 @@ type Realm struct {
 	Tokens []*Token            `gorm:"PRELOAD:false; SAVE_ASSOCIATIONS:false; ASSOCIATION_AUTOUPDATE:false, ASSOCIATION_SAVE_REFERENCE:false"`
 }
 
+func (realm *Realm) EffectiveMFAMode(user *User) *AuthRequirement {
```

Comment:
return by value, saves a bunch of this mucking about with temps.

Reply:
Also saves the indirection in the caller.

Reply:
lol idk how I got so far with these const -> var pointers without thinking "this is easier as a value". Thanks